src/features/offer/components/OfferTags/OfferTags.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useLayoutEffect, useRef, useState } from 'react'
+import { View } from 'react-native'
+import styled from 'styled-components/native'
+
+import { Tag } from 'ui/components/Tag/Tag'
+import { getSpacing } from 'ui/theme'
+
+interface Props {
+  tags: string[]
+}
+
+const MARGIN_BOTTOM_TAGS = getSpacing(2)
+const TAGS_LINES = 2
+
+export function OfferTags({ tags }: Readonly<Props>) {
+  const tagRef = useRef<View>(null)
+  const [tagHeight, setTagHeight] = useState<number>(0)
```

Comment:
J'avais oublié :
```suggestion
  const [tagHeight, setTagHeight] = useState(0)
```
autant laisser TypeScript faire ce qu'il sait faire :D